### PR TITLE
To do checked

### PR DESCRIPTION
**Pull request by supernova-97**
Changes styling of checked to-dos

### CODE REVIEW

**Review by LuYujin9 — APPROVED** [review]
Your app looks cool.

File: components/List/index.js [hunk 1]
```diff
@@ -73,8 +91,21 @@ const StyledDeleteButton = styled.button`
...
+      text-decoration: line-through;
+      opacity: 0.3;
+    `}
 `;
 
+// const StyledTextChecked = styled.p`
+//   margin: 0 15px 0 0;
+//   text-decoration: line-through;
+//   opacity: 0.4;
+// `;
```
Comment (stefankorder) [inline components/List/index.js]: Please delete code you don't use anymore.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Just a little hint.

File: components/List/index.js (hunk 1 above)
Comment (supernova-97) [reply components/List/index.js]: Yep, got rid of it 👍🏻

**Comment by supernova-97** [comment]
Checked to do will be crossed out

**Pull request merged**